bitnami/mariadb-galera/values-production.yaml

```diff
@@ -182,13 +182,14 @@ galera:
   name: galera
 
   ## Bootstraping options
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb-galera#bootstraping
   bootstrap:
     ## Node to bootstrap from, you will need to change this parameter incase you want to bootstrap from other node
     ##
-    bootstrapFromNode: 0
-    ## Force bootstrap
+    bootstrapFromNode:
+    ## Force safe_to_bootstrap in grastate.date file.
     ## This will set safe_to_bootstrap=1 in the node indicated by bootstrapFromNode.
-    forceSafeToBootstrap: false
+    # forceSafeToBootstrap: false
```

Comment:
Why uncommenting this? The same in the `values.yaml`

Reply:
It is true, it would be better to left them uncommeted.